src/js/common/schemaform/definitions.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * These are schema definitions for some common form fields
+ */
+
+export const fullName = {
+  type: 'object',
+  title: '',
+  required: ['first', 'last'],
+  properties: {
+    first: {
+      type: 'string',
+      title: 'First name'
+    },
+    middle: {
+      type: 'string',
+      title: 'Middle name'
+    },
+    last: {
+      type: 'string',
+      title: 'Last name',
+      minLength: 1,
+      maxLength: 30
+    },
+    suffix: {
+      type: 'string',
+      title: 'Suffix',
+      'enum': [
+        '',
+        'Jr.',
+        'Sr.',
+        'II',
+        'III',
+        'IV'
+      ]
+    }
+  }
+};
+
+export const ssn = {
+  title: 'Social security number',
+  type: 'string',
+  pattern: '^([0-9]{3}-[0-9]{2}-[0-9]{4}|[0-9]{9})$'
```

Comment:
This is used in the JSON schema validation, which the library runs internally. The stuff in validations.js runs in addition to that validation, so that we can do checks that aren't possible in JSON schema.

I'm not sure what the best way to deal with the two separate sets of validations is. If we didn't care about being in sync with JSON schema validation on the backend, we might want to just turn off the JSON schema validation and use our own.